src/data/contributors.dev.json

```diff
@@ -0,0 +1,598 @@
+[
```

Comment:
A snapshot of real contributor data as of today. This file will be used for development purposes only. It gets copied to `contributors.json` so that the app behaves normally when running locally.

For builds, we still retrieve real contributor data at the time of build.